tests/run-make/core-ffi-typecheck/rmake.rs

```diff
@@ -0,0 +1,129 @@
+use std::fs;
+
+use run_make_support::{clang, rfs, rustc};
+
+fn main() {
+    // Get list of all targets
+    let targets = get_target_list();
+
+    // For each target:
+    for target in targets.lines() {
+        // Run clang to get target info
+
+        //TODO: someone please test riscv targets as clang on my machine does not have that llvm
+        //target
+
+        if target.starts_with("riscv") {
+            continue;
+        }
+        if target.starts_with("wasm") {
+            continue;
+        }
+        if target.starts_with("xtensa") {
+            continue;
+        }
```

Comment:
Could this instead be a `const SKIPPED_TARGETS: &[&str] = &[...]` at the top of the file, with the full exact target names? So we don't cross off more than is needed.